src/librbd/cache/rwl/LogEntry.h

```diff
@@ -30,6 +31,9 @@ class GenericLogEntry {
   virtual ~GenericLogEntry() { };
   GenericLogEntry(const GenericLogEntry&) = delete;
   GenericLogEntry &operator=(const GenericLogEntry&) = delete;
+  virtual bool can_flush() {
```

Comment:
Nit: The `flush` in `can_flush()` and `flush()` below seems a little confusing / overloaded terminology. Perhaps `can_writeback` / `writeback`?

Reply:
@dillaman use can_writeback() instead. Done.